packages/animations/example/lib/shared_axis_transition/shared_axis_transition.dart

```diff
@@ -0,0 +1,252 @@
+import 'package:flutter/material.dart';
+import 'package:animations/animations.dart';
+
+/// The demo page for [SharedAxisPageTransitionsBuilder].
+class SharedAxisTransitionDemo extends StatefulWidget {
+  @override
+  _SharedAxisTransitionDemoState createState() {
+    return _SharedAxisTransitionDemoState();
+  }
+}
+
+class _SharedAxisTransitionDemoState extends State<SharedAxisTransitionDemo> {
+  final Key coursePageKey = const Key('Course Page');
+  final Key signInPageKey = const Key('Sign In Page');
+
+  SharedAxisTransitionType transitionType = SharedAxisTransitionType.horizontal;
+  bool isLoggedIn = false;
+
+  void updateTransitionType(SharedAxisTransitionType newType) {
+    setState(() {
+      transitionType = newType;
+    });
+  }
+
+  void toggleLoginStatus() {
+    setState(() {
+      isLoggedIn = !isLoggedIn;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      resizeToAvoidBottomPadding: false,
+      appBar: AppBar(title: const Text('Shared Axis Transition')),
+      body: LayoutBuilder(
```

Comment:
Do you need the LayoutBuilder? Would a SizedBox.expanded do? 

https://master-api.flutter.dev/flutter/widgets/SizedBox/SizedBox.expand.html

Reply:
Wondering if you even need the SizedBox.expanded. I would expect that the body property already enforces this?

Reply:
Actually, I forgot that we had an `Expanded` widget, and that works perfectly. I'll remove the LayoutBuilder and the `SizedBox` widget in favor of that!